package.json

```diff
@@ -342,4 +332,4 @@
         "xmldom": "^0.6.0",
         "xpath": "0.0.32"
```

Comment:
Can you propose one combined PR for all of your changes?

Reply:
I understand this is extra work. Even for me.
It is also for you easier to review.
But I will propose a single PR once you have reviewed them all.
Just put a comment with "OK to merge them all" or something and I will merge all PRs into a single